sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_01_wo_or_replace/cases/01_01_01-01_normal_basic.sql

```diff
@@ -0,0 +1,17 @@
+--+ server-message on
```

Comment:
--+ server-output on
--+ server-message on

which one, would be used?

Reply:
Only the server-message will be used. Please note that the hint is going to be changed from 'server-output' to 'server-message'  in the PR (https://github.com/CUBRID/cubrid-testtools/pull/662)
```
--+ server-message on
```